src/components/StyledDivider.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const StyledDivider = styled.div`
```

Comment:
Maybe an `<hr />` tag is also working here? Not sure if it has semantically advantages. 🙂

Reply:
True, it has the `aria` role of *separator*. Thanx :)